rest_framework_json_api/settings.py

```diff
@@ -6,14 +6,19 @@
 
 from django.conf import settings
 from django.core.signals import setting_changed
+import warnings
 
 JSON_API_SETTINGS_PREFIX = 'JSON_API_'
 
+RELATIONS_RENDERING_STRATEGY = 'RELATIONS'
+ATTRIBUTE_RENDERING_STRATEGY = 'ATTRIBUTE'
+
 DEFAULTS = {
     'FORMAT_FIELD_NAMES': False,
     'FORMAT_TYPES': False,
     'PLURALIZE_TYPES': False,
     'UNIFORM_EXCEPTIONS': False,
+    'NESTED_SERIALIZERS_RENDERING_STRATEGY': RELATIONS_RENDERING_STRATEGY
```

Comment:
up to this point all DJA settings are booleans. And I don't think we will have a third strategy and this option will be removed in the future anyhow. So I think it is better to have a boolean setting instead of arbitrary values. It will also be easier to configure this way for the user.

Such setting could be called `SERIALIZE_NESTED_SERIALIZERS_AS_JSON` or something.